ws-nextjs-app/pages/[service]/new_live/[id]/Post/index.tsx

```diff
@@ -157,17 +159,22 @@ const Post = ({ post }: { post: PostType }) => {
 
   return (
     <div css={styles.postContainer}>
```

Comment:
To meet the screen reader UX I believe we are missing an article element? Should we output this div as article or the next div?